v20.2/admin-ui-sessions-page.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Sessions Page
+toc: true
+---
+
+{{site.data.alerts.callout_info}}
+On a secure cluster, this area of the Admin UI can only be accessed by a user with [`VIEWACTIVITY`](authorization.html#assign-privileges) privileges. Note that non-`admin` users will see only their own sessions, while `admin` users see sessions for all users.
+{{site.data.alerts.end}}
+
+The **Sessions** page of the Admin UI provides details of all open sessions on the cluster.
+
+To view this page, [access the Admin UI](admin-ui-overview.html#admin-ui-access) and click **Sessions** in the left-hand navigation.
+
+## Sessions list
+
+Use the **Sessions** list to see the open sessions on the cluster. This includes active and idle sessions. 
+
+{{site.data.alerts.callout_info}}
+An session is *active* if it has an open transaction, and *idle* if it has no open transaction. Active sessions consume [hardware resources](recommended-production-settings.html#hardware).
+{{site.data.alerts.end}}
+
+- If an active session also has an open transaction, the most recent SQL statement will be displayed in the **Statement** column.
+- To view [details of a session](#session-details), click on the **Session Age**.
+
+<img src="{{ 'images/v20.2/admin_ui_sessions.png' | relative_url }}" alt="CockroachDB Admin UI Database Tables View" style="border:1px solid #eee;max-width:100%" />
+
+The following are displayed for each active session:
+
+Parameter | Description
+--------- | -----------
+Session Age | Amount of time the session has been open.
+Txn Duration | Amount of time the transaction has been active, if there is an open transaction.
```

Comment:
Not your problem - but hm I really wish we didn't have both Age and Duration here... I'll try to make this better